README.md

```diff
@@ -42,13 +44,10 @@ npm run build
 ##### Developing:
 Run each of these commands in separate terminals:
 
-To rebuild your js every time you save:
+To rebuild your js every time you save and to serve your page at [http://localhost:3456](http://localhost:3456):
```

Comment:
Can we shorten this a bit? Something like: 
To run a development build of the project at https://localhost:3456

Reply:
Of course 👩‍💻, thanks for your input 🤗.